style.css

```
@@ -1,8 +1,24 @@
+@import "common.css";
+
 @media(max-width:1920px){
     p{
         white-space: nowrap;
     }
 }
+@media(max-width:1920px){
```

💊 제안
각 항목을 구분할 수 있게 띄어쓰는 것을 추천드려요~

```suggestion
@media (max-width: 1920px) {
```